guides/source/i18n.md

```diff
@@ -791,7 +791,7 @@ Alternatively, the separate gem [rails-i18n](https://github.com/svenfuchs/rails-
 
 ### Setting and Passing a Locale
 
-The locale can be either set pseudo-globally to `I18n.locale` (which uses `Thread.current` like, e.g., `Time.zone`) or can be passed as an option to `#translate` and `#localize`.
+The locale can be either set pseudo-globally to `I18n.locale` (which uses `Thread.current` like `Time.zone`) or can be passed as an option to `#translate` and `#localize`.
```

Comment:
According to [this](https://ell.stackexchange.com/questions/222059/is-it-using-like-e-g-seen-as-incorrect), "like" and "e.g." are interchangable, therefore I removed e.g. Being relatively new to Rails I don't 100% understand what that sentence tells me, the meaning may be lost by that change.

Reply:
Here, "like" means "in the same way as":

```suggestion
The locale can be either set pseudo-globally to `I18n.locale` (which uses `Thread.current` in the same way as, e.g., `Time.zone`) or can be passed as an option to `#translate` and `#localize`.
```

You could also swap "e.g." with "for example", but I'm not sure whether that is more clear?

```suggestion
The locale can be either set pseudo-globally to `I18n.locale` (which uses `Thread.current` in the same way as, for example, `Time.zone`) or can be passed as an option to `#translate` and `#localize`.
```